## Hypothesis Testing Assignment
### Cutlets.csv

Assume Null Hypothesis and Alternative hypothesis
Ho = μ1 = μ2 (There is no significant diffenrece between diameter of two sample of cutlet)
Ha= μ1 !=μ2 (There is significant difference between diameter of two samples of cutlet)

For this example we will use two sample two tail test

In [2]:
#Importing the libraries
import pandas as pd 
import numpy as np 
import scipy
from scipy import stats

In [3]:
df = pd.read_csv(r'C:\Users\91901\Desktop\Cutlets.csv')

In [4]:
df.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

In [6]:
group_A = pd.Series(df.iloc[:,0]) #selecting column 1 
group_A.head()

0    6.8090
1    6.4376
2    6.9157
3    7.3012
4    7.4488
Name: Unit A, dtype: float64

In [7]:
group_B = pd.Series(df.iloc[:,1]) #selecting column 2 
group_B.head()

0    6.7703
1    7.5093
2    6.7300
3    6.7878
4    7.1522
Name: Unit B, dtype: float64

In [8]:
# Two Sample Two Tail Test
stats.ttest_ind(group_A,group_B)

Ttest_indResult(statistic=0.7228688704678061, pvalue=0.4722394724599501)

As P value is 0.4722394724599501 > 0.05 we can reject the alternative hypothesis.
We can say that there is no significant difference between diameters of cutlet from two units